pages/list/[id].js

```diff
@@ -21,37 +21,50 @@ export default function List() {
   const router = useRouter();
   const { id } = router.query;
 
-  const lists = useListsStore((state) => state.lists);
-  const categories = useCategoriesStore((state) => state.categories);
+  const lists = useStore((state) => state.lists);
+  const categories = useStore((state) => state.categories);
 
   let list;
   if (lists) {
     list = lists.find((list) => list.id === id);
   }
 
-  let filteredCategories;
+  let activeCategories;
+  let inactiveCategories;
   if (list) {
-    filteredCategories = categories.filter((category) =>
-      category.listId === list.id ? category : ""
+    activeCategories = categories.filter(
+      (category) => category.listId === list.id
+    );
+    inactiveCategories = categories.filter(
+      (category) => category.listId === list.id && category.hasDisabledItems
     );
   }
 
-  if (!list) return;
+  if (!activeCategories && !inactiveCategories) return;
 
   return (
     <>
       <Layout>{list.name}</Layout>
       <ListMain>
         <CategoriesSection>
-          {!filteredCategories.length && (
+          {!activeCategories.length && (
             <Info>Derzeit noch keine Einträge oder Kategorien vorhanden</Info>
           )}
-          {filteredCategories &&
-            filteredCategories.map((category) => (
+          {activeCategories &&
+            activeCategories.map((category) => (
+              <DynamicCategory
+                key={category.id}
+                category={category}
+                listId={id}
+              />
+            ))}
+          {inactiveCategories &&
```

Comment:
unnecessary logical and operators, since both variables used here are arrays and all arrays are truthy values always.